ckanext/harvest/commands/harvester.py

```diff
@@ -20,6 +20,9 @@ class Harvester(CkanCommand):
       harvester rmsource {id}
         - remove (inactivate) a harvester source
 
+      harvester clearsource {id}
+        - clear a harvester source (delete all associtated datasets)
```

Comment:
I think it would be good to be clearer and use the language in the logic function here (and in the README): "Clears all datasets, jobs and objects related to a harvest source, but keeps the source itself."